internal/pkg/deploy/cloudformation/stack/rd_web_svc.go

```diff
@@ -62,12 +62,16 @@ func (s *RequestDrivenWebService) Template() (string, error) {
 	if err != nil {
 		return "", err
 	}
-
+	publishers, err := convertPublish(s.manifest.Publish, s.rc.AccountID, s.rc.Region, s.app.Name, s.env, s.name)
+	if err != nil {
+		return "", fmt.Errorf("convert the publish field for service %s: %w", s.name, err)
+	}
 	content, err := s.parser.ParseRequestDrivenWebService(template.ParseRequestDrivenWebServiceInput{
 		Variables:         s.manifest.Variables,
 		Tags:              s.manifest.Tags,
 		NestedStack:       outputs,
 		EnableHealthCheck: !s.healthCheckConfig.IsEmpty(),
+		Publish:           publishers,
```

Comment:
Nothing to do now but we should message very clearly in `svc deploy` if the service type is RD Web Service that only `Worker Services` deployed on ECS can currently read from these topics.